### ISSUE TITLE
Performance drop when increasing the input sequence length with Longformer.

### ISSUE THREAD

**Issue by ValkyriaLenneth** [allenai/allennlp#5479]
Hello, I am using ALLENNLP and ALLENNLP-MODELS to evaluate the performance of Longformer(https://huggingface.co/transformers/model_doc/longformer.html) on Chinese Coreference task.
The command I used is `allennlp train`. 
The training process is well when setting the max input sequence to 512.
However, when I set the max sequence length to 1024, the performance of Longformer model decreases. 

Performance of Longformer model(512):
```
"best_validation_coref_precision": 0.7030205732402074,
  "best_validation_coref_recall": 0.6367544322591994,
  "best_validation_coref_f1": 0.6682187062522592,
  "best_validation_mention_recall": 0.9151096382993725,
  "best_validation_loss": 115.7285145906786
```

Performance of Longformer model(1024):
```
"best_validation_coref_precision": 0.6866745335175204,
  "best_validation_coref_recall": 0.6188566654354536,
  "best_validation_coref_f1": 0.650972266138495,
  "best_validation_mention_recall": 0.9184939716562082,
  "best_validation_loss": 122.35920043677415
```

I cannot understand why increasing the input sequence length will lead to worse performance.
How could I check the correctness of input attention mask of ALLENNLP, maybe the reason is this?

**Comment by epwalsh** [comment]
Hi @ValkyriaLenneth, are you using the same hyperparameters for the 1024 model? While it's possible that there's a bug that's causing this performance drop, I think it's more likely that you just need to adjust the hyperparameters.

**Comment by epwalsh** [comment]
But to debug the model, I would run it on CPU with [pdb](https://docs.python.org/3/library/pdb.html).
